pkspace/utils/loaders.py

```diff
@@ -0,0 +1,105 @@
+import glob
+import json
+import os
+import xml.etree.ElementTree
+import cv2
+import numpy as np
+
+
+class Loader:
+    def load(self, path, split):
+        pass
+
+
+class PKspaceLoader(Loader):
+    def __get_spaces__(self, src, desc):
+        finsize = (80, 80)
+        spaces = []
+        answers = []
+        for space in desc.get('spots'):
```

Comment:
@rstevanak I can see why you may want to do `desc.get('sports')` here -- if the `sports` key does not exist in `desc` you would get `None` in return and the whole loop won't get executed. 

I do not really see why would you do the same on the lines below: if there is no `points` for a specific space, your `points` variable will be set to `None` and the code below that makes use of it will play with it just as if nothing happened. it will of course produce wrong results, but you'll notice that much later. I believe it would be better to let it trigger `KeyError` or do not let the code below execute if either `points` or `angle` is `None`. in case of `angle`, you may want to define some default, but I do not see what else could be done for `points`.